### ISSUE TITLE
Use database level information from metadata.json on index.html

### ISSUE THREAD

**Issue by lagolucas** [simonw/datasette#646]
Did a search on the issues here and didn't find anything related to what I want.

I want to have information that is on the database level of the JSON like title, source and source_url, and use it on the index page.

I tried some small tweaks on the python and html files, but failed to get that result.

Is there a way? Thanks!

**Comment by simonw** [comment]
I don't think this is possible at the moment but you're right, it totally should be.

**Comment by lagolucas** [comment]
Thanks for the reply.

Will try to implement that on my end, if I have any success I will post here/ make a pull request.

**Comment by lagolucas** [comment]
> I don't think this is possible at the moment but you're right, it totally should be.

Just give me a headsup if you think you can do that quickly. I am trying to implement it with very little knowledge of how datasette works, so it will take loads of time.
